0000-template.md

```diff
@@ -1,15 +1,22 @@
 # Feature name
 
 * Proposal: [SE-NNNN](NNNN-filename.md)
-* Author: [Swift Developer](https://github.com/swiftdev)
-* Status: **Awaiting review**
+* Authors: [Swift Developer](https://github.com/swiftdev), [Swift Developer](https://github.com/swiftdev)
```

Comment:
The multiple "\[Swift Developer\]" looks a bit weird, since they are the same.  Maybe "Author 1", "Author 2"?